src/components/add-account-dialog/add-account-dialog.js

```diff
@@ -65,38 +84,56 @@ export class AddAccountDialogComponent {
   }
 
   handleOk() {
-    this.props.onAddAccountConfirm({
-      name: this.accountNameInput.value,
-      balance: parseInt(this.balanceNameInput.value),
-      type: this.accountType.innerText,
-      currency: this.accountCurrency.innerText,
-      id: Math.random()
-        .toString(36)
-        .substring(2)
-    });
-    this.clean();
+    if (this.accountTitle.innerText === "Add account") {
```

Comment:
you can set in `showDialogEdit ` method property on instance like `this.isEditMode` -- and here you will check it